### PR TITLE
Feature/to plato scene

### PR DESCRIPTION
**Pull request by bdice**
## Description
Provides a conversion from a `Frame` object to a `plato.draw.Scene` object containing the shapes determined from the frame's shape definitions. Requires https://github.com/glotzerlab/plato/pull/23 for the `Box` primitive.

## Motivation and Context
This is a convenience method for visualization of particle systems. It will add `plato` as a soft dependency (the function just fails if the user doesn't have `plato`).

## How Has This Been Tested?
Tests have been added, which produce sample renderings named like `tests/files/shapes/*.png`.

The tests don't run on CI because I didn't want to add both `plato` and its backends to the testing dependencies without a second opinion. If the maintainers think it would be appropriate, I can add some artifact-based tests (which must be viewed manually on CircleCI). This is how `plato` gets tested currently.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds or improves functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update

## Checklist:
- [x] I have read the [**CONTRIBUTING**](https://github.com/glotzerlab/garnett/blob/master/CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [x] I have updated the documentation (if relevant).
- [x] I have added tests that cover my changes (if relevant).
- [x] All new and existing tests passed.
- [x] I have updated the [Changelog](https://github.com/glotzerlab/garnett/blob/master/changelog.rst).

### CODE REVIEW

**Comment by bdice** [comment]
## Questions
- Should the default shape be gray or blue or something else?

## Screenshots

## convex_polyhedron.gsd
image

## convex_spheropolygon.gsd
image

File: garnett/trajectory.py [hunk 1]
```diff
@@ -350,6 +354,123 @@ def copyto_snapshot(self, snapshot):
...
+        :type backend: str
+        :param scene: Scene object to render into. By default, a new scene is
+                      created.
+        :type scene: :class:`plato.draw.Scene`
+        """
+        try:
+            import importlib
+            backend = importlib.import_module('plato.draw.{}'.format(backend))
+        except ImportError:
+            raise
```
Comment (lyrivera) [inline garnett/trajectory.py]: Can you add an explicit message here?

**Review by lyrivera — APPROVED** [review]
Left one minor comment. Except that, everything looks good to me.

**Comment by lyrivera** [comment]
> ## Questions
> * Should the default shape be gray or blue or something else?
> 
> ## Screenshots
> ## convex_polyhedron.gsd
> image
> 
> ## convex_spheropolygon.gsd
> image

I think grey is fine.

**Pull request merged**